libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -37,10 +37,13 @@ import { StripeConfig } from './stripe.config';
 export class StripeClient {
   private readonly stripe: Stripe;
   constructor(private stripeConfig: StripeConfig) {
-    this.stripe = new Stripe(this.stripeConfig.apiKey, {
-      apiVersion: '2024-04-10',
-      maxNetworkRetries: 3,
-    });
+    this.stripe = new Stripe(
+      this.stripeConfig.apiKey || 'api_key_placeholder',
```

Comment:
Is the placeholder necessary?

Reply:
As discussed, I've added the following comment in the code
```
// 'api_key_placeholder' is currently needed during build time
// Should be able to remove in the next version
// https://github.com/stripe/stripe-node/issues/2207
```